.github/workflows/continuous-integration-workflow.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       run: docker build --target test --tag do-me:test .
 
     - name: Launch Docker container do-me:test and run unit tests
-      run: docker run -e TRELLO_KEY=FAKE_KEY -e TRELLO_TOKEN=FAKE_TOKEN -e TRELLO_BOARD_NAME=ToDoApp -i do-me:test todo_app/tests
+      run: docker run -e MONGODB_CONNECTIONSTRING=FAKE_CONNECTIONSTRING -e MONGODB_DATABASE=doMeDatabase -i do-me:test todo_app/tests
```

Comment:
Your `.env.test` file is checked in since it only contains fake values, so one option to save having to update this is to use it here rather than explicit environment variables